Source/Scene/CameraEventAggregator.js

```diff
@@ -262,7 +262,8 @@ define([
         }
         //>>includeEnd('debug');
 
-        this._eventHandler = new ScreenSpaceEventHandler(canvas);
+        this._eventHandler = new ScreenSpaceEventHandler(canvas, false);
```

Comment:
I fixed the warning. A `ScreenSpaceEventHandler` was being created by the `CesiumWidget`.